.github/workflows/deploy-image-test.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy Image to Docker Hub
+
+on:
+  pull_request
+
+jobs:
+  deploy_image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: ./Dockerfile
+          push: false # true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/fosslight:${{ github.event.release.tag_name }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

Comment:
@soimkim, `deploy-to-docker-image.yml` is a test workflow. You should **REMOVE** this file after test.

```suggestion

```